lib/src/file_io_desktop_and_mobile.dart

```diff
@@ -5,14 +5,20 @@ import 'package:path_provider/path_provider.dart';
 
 bool get isMacOS => Platform.isMacOS;
 
-Future<void> saveFontToDeviceFileSystem(String name, List<int> bytes) async {
-  final file = await _localFile(name);
+Future<void> saveFontToDeviceFileSystem(
+  String name,
```

Comment:
nit: use named params here

Reply:
nit: make filehash las named param in list

Reply:
woops, will fix